extensions/blocks/contact-form/components/jetpack-field-consent.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { BaseControl, PanelBody, SelectControl } from '@wordpress/components';
+import { InspectorAdvancedControls, InspectorControls } from '@wordpress/block-editor';
+import { withInstanceId } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import JetpackFieldLabel from './jetpack-field-label';
+import JetpackFieldWidth from './jetpack-field-width';
+import JetpackFieldCss from './jetpack-field-css';
+
+const JetpackFieldConsent = ( {
+	id,
+	instanceId,
+	width,
+	consentType,
+	implicitConsentMessage,
+	explicitConsentMessage,
+	setAttributes,
+} ) => {
+	return (
+		<BaseControl
+			id={ `jetpack-field-consent-${ instanceId }` }
+			className="jetpack-field jetpack-field-consent"
+			label={
+				<>
+					{ consentType === 'explicit' && (
+						<input className="jetpack-field-consent__checkbox" type="checkbox" disabled />
+					) }
+					<JetpackFieldLabel
+						required={ false }
+						label={
+							{
+								implicit: implicitConsentMessage,
+								explicit: explicitConsentMessage,
+							}[ consentType ] ?? ''
+						}
+						setAttributes={ setAttributes }
+						labelFieldName={ `${ consentType }ConsentMessage` }
+						placeholder={ sprintf( __( 'Add %s consent message…', 'jetpack' ), consentType ) }
```

Comment:
Could you add a translators comment to help folks with that string? Something like:

```suggestion
						placeholder={ sprintf(
							/* translators: placeholder is a type of consent: implicit or explicit */
							__( 'Add %s consent message…', 'jetpack' ),
							consentType
						) }
```

Reply:
Done, Thanks for the tips, wasn't aware.